code/game/machinery/computer/depot.dm

```diff
@@ -84,17 +72,54 @@
 /obj/machinery/computer/syndicate_depot/proc/disable_special_functions()
 	return
 
-/obj/machinery/computer/syndicate_depot/Topic(href, href_list)
+/obj/machinery/computer/syndicate_depot/ui_interact(mob/user, ui_key = "main", datum/tgui/ui = null, force_open = TRUE, datum/tgui/master_ui = null, datum/ui_state/state = GLOB.default_state)
+	ui = SStgui.try_update_ui(user, src, ui_key, ui, force_open)
+	if(!ui)
+		ui = new(user, src, ui_key, "ComputerSimple",  name, window_width, window_height, master_ui, state)
```

Comment:
```suggestion
		ui = new(user, src, ui_key, "SyndicateComputerSimple",  name, window_width, window_height, master_ui, state)
```
UI should be prefixed with `Syndicate`. `ComputerSimple` is too generic and people may end up trying to use it to make on-station things.